src/shmem_comm.h

```diff
@@ -25,22 +25,6 @@
 #include "shmem.h"
 #include "shmemx.h"
 
-#include "shmem_atomic.h"
```

Comment:
I think we need this header for `shmem_internal_membar_load`, which is causing the Travis failure in the CMA row.